packages/katello/katello/katello.spec

```diff
@@ -81,10 +80,9 @@ gzip -f9 %{_sourcedir}/katello-change-hostname.8
 mkdir -p %{buildroot}/%{_mandir}/man8
 
 #copy cron scripts to be scheduled
-install -d -m0755 %{buildroot}%{_sysconfdir}/cron.weekly
+install -d -m0755 %{buildroot}%{_sysconfdir}/cron.d
 install -d -m0755 %{buildroot}%{_sysconfdir}/cron.daily
-install -m 755 %{SOURCE3} %{buildroot}%{_sysconfdir}/cron.weekly/katello-remove-orphans
-install -m 755 %{SOURCE10} %{buildroot}%{_sysconfdir}/cron.weekly/katello-clean-empty-puppet-environments
+install -m 644 %{SOURCE18} %{buildroot}%{_sysconfdir}/cron.d/katello
```

Comment:
You can remove the `mkdir` by using `install -D`

Reply:
Not sure what `mkdir` you're referring to? the `install -m` here sets the permissions.

Reply:
Above you have `install -d %{_sysconfdir}/cron.d` but if you use `install -D` instead of `install` here that should be redundant I think. But feel free to keep it the way it is.